contexts/network-settings.tsx

```diff
@@ -273,36 +281,72 @@ export const NetworkSettingsProvider = ({ children }) => {
 
     return dao;
   }
+  
+  const cleanStorage = () => localStorage.removeItem(`${localStorageKey}:${wallet.address}`)
+
+  async function loadDefaultSettings(){
+    const defaultState = DefaultNetworkSettings;
 
-  function handlerDefaultSettings(){
-    setFields('settings.theme.colors', DefaultTheme())
+    const [tokensLockedInRegistry, registryCreatorAmount] = await Promise.all([
+        DAOService.getTokensLockedInRegistryByAddress(wallet.address),
+        DAOService.getRegistryCreatorAmount()
+    ])
     
-    setFields('settings.parameters', {
-    draftTime: {
-      value: DEFAULT_DRAFT_TIME,
-      validated: undefined
-    },
-    disputableTime: {
-      value: DEFAULT_DISPUTE_TIME,
-      validated: undefined
-    },
-    percentageNeededForDispute: {
-      value: DEFAULT_PERCENTAGE_FOR_DISPUTE,
-      validated: undefined
-    },
-    councilAmount: {
-      value: DEFAULT_COUNCIL_AMOUNT,
-      validated: undefined
-    },
-    validated: undefined
-    })
+    defaultState.tokensLocked = {
+      amount: '0',
+      locked: BigNumber(tokensLockedInRegistry).toString(),
+      needed: BigNumber(registryCreatorAmount).toString(),
```

Comment:
is better use `toFixed` instead of `toString`, in some cases `toString` can result in a scientific notation string